php/meterpreter/ext_server_stdapi.php

```diff
@@ -1266,6 +1266,37 @@ function stdapi_registry_set_value($req, &$pkt) {
 }
 }
 
+if (!function_exists('stdapi_net_config_get_arp_table')) {
+if (is_linux()) {
+    register_command('stdapi_net_config_get_arp_table');
+}
+function stdapi_net_config_get_arp_table($req, &$pkt) {
+    if !(is_linux()) {
+        return ERROR_FAILURE;
+    }
+    $content = file_get_contents('/proc/net/arp');
+    if ($content === false) {
+      return ERROR_FAILURE;
+    }
+    $lines = explode(PHP_EOL, $content);             
+    array_shift($lines); // first line is the header of the array
+    foreach($lines as $line) { 
+      if ($line == '') continue; 
+      $v = preg_split('/\s+/', $line); 
+      $ip = $v[0]; 
+      $mac = $v[3]; 
+      $iface = $v[5]; 
+      my_print("arp line: $ip $mac $dev"); 
+      $arp_tlv  = tlv_pack(create_tlv(TLV_TYPE_IP, inet_pton($ip)));
+      $arp_tlv .= tlv_pack(create_tlv(TLV_TYPE_MAC_ADDRESS, $mac));
```

Comment:
The MAC address needs to be packed as bytes.

```suggestion
      $arp_tlv .= tlv_pack(create_tlv(TLV_TYPE_MAC_ADDRESS, pack("H*", str_replace(':', '', $mac))));
```